src/chrome/content/rules/orange.fr.xml

```diff
@@ -0,0 +1,23 @@
+<ruleset name="Orange France (Partial)" >
+  <target host="boutique.orange.fr" />
+  <target host="id.orange.fr" />
+  <target host="iz.orange.fr" />
+  <target host="web.cloud.orange.fr" />
+  <target host="assistance.orange.fr" />
+  <target host="jeu.orange.fr" />
+  <target host="*.espaceclientv*.orange.fr" />
```

Comment:
Given there are multiple targets with misplaced asterisks, I assume that you mean to target $(subdomain).*\.orange\.fr. Targets accept only wildcards or only literal text for any given domain level.